pkg/middleware/auth_test.go

```diff
@@ -150,6 +155,116 @@ func TestAuth_Middleware(t *testing.T) {
 	}
 }
 
+func TestRoleAppPluginAuth(t *testing.T) {
+	t.Run("Verify user's role when requesting app route which requires role", func(t *testing.T) {
```

Comment:
Could be nice to have RBAC tests as well, wdyt?

Reply:
Looks good to me :100: . I tried to see if we could have used the real accesscontrol service in the test (`acimpl.ProvideAccessControl(sc.cfg)`) but it does an import cycle.